lms/templates/commerce/checkout_cancel.html

```diff
@@ -8,6 +12,6 @@
 
 <section class="container">
     <h1>${_("Checkout Cancelled")}</h1>
-    ${ _(u"Your transaction has been cancelled. If you feel an error has occurred, contact {email}.").format(
-    email="<a href=\"mailto:{email}\">{email}</a>".format(email=payment_support_email)) }
+    ${ Text(_(u"Your transaction has been cancelled. If you feel an error has occurred, contact {email}.")).format(
+    email=HTML("<a href=\"mailto:{email}\">{email}</a>").format(email=payment_support_email)) }
```

Comment:
Nit: can you clean up the indentation (I realize it wasn't you who wrote it this way). Thanks.